packages/components/data-table-manager/src/data-table-manager.story.js

```diff
@@ -350,6 +477,8 @@ storiesOf('Components|DataTable', module)
       'showColumnManagerConfirmationButtons',
       false
     );
+    // TODO - remove when nexted rows are implemented
+    const debug = boolean('logCustomSettings', false);
```

Comment:
This is solely for storybook and testing purpose. Will be helpful when implementing the nested rows. It is currently a toggle in storybook to help see the data passed in the customSettingsPayload